cores/esp32/Arduino.h

```diff
@@ -168,12 +174,13 @@ void shiftOut(uint8_t dataPin, uint8_t clockPin, uint8_t bitOrder, uint8_t val);
 #include "Esp.h"
 #include "esp32/spiram.h"
 
+// Use float-compatible stl abs() and round(), we don't use Arduino macros to avoid issues with the C++ libraries
```

Comment:
Does this comment about abs() and round() conflict with declaration of round() above?

Reply:
ooppss... I left it behind!
yes, `#define round` will conflict with `std::round`, as the compiler will try to apply the macro.
Thanks!